.github/workflows/build.yml

```diff
@@ -0,0 +1,22 @@
+on:
+  # Trigger analysis when pushing in master or pull requests, and when creating
+  # a pull request.
+  push:
+    branches:
+      - pre-release
```

Comment:
add master in here too for when we go back to using master?

Reply:
sure, I was just exploring here a bit with pre-release
We might use the bot instead of a workflow here